tests/test_granule_to_sns.py

```diff
@@ -0,0 +1,108 @@
+import boto3
+import pytest
+
+from granule_to_sns import generate_message, granule_to_sns
+
+
+@pytest.fixture
+def get_event():
```

Comment:
Fixtures names should generally just be nouns. See the examples in the docs: https://docs.pytest.org/en/8.2.x/explanation/fixtures.html#about-fixtures

```suggestion
def event():
```